Update CONTRIBUTING.md (#3540)
geosolutions-it/MapStore2: CONTRIBUTING.md

## Improving Documentation

> All MapStore components (i.e. plugins, actions, reducers etc) need to be properly documented in their JS documentation, please follow the required **JSDoc** syntax looking at the online documentation available in the [JSDoc web site](http://usejsdoc.org/index.html) for more details. 

Reviewer: ```suggestion
All MapStore objects (i.e. components, plugins, actions, reducers etc) should be properly documented in their JS documentation, please follow the required **JSDoc** syntax looking at the online documentation available in the [JSDoc web site](http://usejsdoc.org/index.html) for more details. 
```

> Each new feature/tool in MapStore needs to be documented also in the [User Guide](https://mapstore2.readthedocs.io/en/latest/user-guide/home-page/) in order to explain the involved functionalities and illustrate how it works. 

Reviewer: ```suggestion
Each new feature/tool in MapStore should be documented also in the [User Guide](https://mapstore2.readthedocs.io/en/latest/user-guide/home-page/) in order to explain the involved functionalities and illustrate how it works. 
```

> All new front-end technologies, development procedures, best practices and guidelinees on the involved components in MapStore need to be properly documented too: the [Developers Guide](https://mapstore2.readthedocs.io/en/latest/developer-guide/) must be kept up-to-date for this.

Reviewer: I don't think this is relevant for contributors. It's more a GeoSolutions internal practice.
